twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/slack/SlackNotificationTemplate.java

```diff
@@ -39,9 +40,10 @@ public SlackNotificationTemplate(
         this.stateRegex = stateRegex;
     }
 
-    public Optional<String> match(String user, String event, String state)
+    public Optional<String> match(String user, String principal, String event, String state)
     {
         if (userRegex.map(regex -> regex.matcher(user).matches()).orElse(true) &&
+                principal.map(regex -> regex.matcher(principal).matches()).orElse(true) &&
```

Comment:
Should this be `principalRegex.map(regex -> regex.matcher(principal).matches()).orElse(true)`?

Reply:
Fixed